src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -1,74 +1,81 @@
 :host {
-  @extend %component-host;
-  @apply items-stretch
+	@extend %component-host;
```

Comment:
We should probably replace this with any relevant styles from this partial.

Reply:
Host partial is [here](https://github.com/Esri/calcite-components/blob/master/src/assets/styles/_host.scss).

Reply:
Handling in a separate PR.